lib/Source.ml

```diff
@@ -367,3 +367,8 @@ let is_quoted_string t loc =
       | _ -> false)
   in
   not (List.is_empty toks)
+
+let is_before_match_keyword t kwd ~exp_loc ~cmt_loc =
```

Comment:
This is not only used for `match`, can we find a better name ?

Reply:
I've changed the function, the new one is now called `loc_of_keyword`